01-basic-java-syntax.md

```diff
@@ -96,47 +125,67 @@ A class variable is also known as a static field. An instance/member variable is
 new <<Call to Class Constructor>>;
 ```
 
-The new operator is followed by a call to the constructor of the class whose instance is being created. The new operator creates an instance of a class by allocating the memory on heap.
-*I don't really need to worry about memory allocation at the moment because Java handles this with the garbage collector.*
+The new operator is followed by a call to the constructor of the class whose instance is being created. The `new` operator creates an instance of a class (by allocating the memory on heap). *no need to think about memory allocation at the moment because Java handles this automagically.*
```

Comment:
**automagically** - if this is a typo - then it's probably the greatest one I've ever seen. I'm going to use this word from now on!

Reply:
Wish I could take credit for it.
https://en.oxforddictionaries.com/definition/automagically